src/icons/lanaCardATM.svg

```diff
@@ -0,0 +1,41 @@
+<svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this comment is not about this line but about the file name itself: please rename to: `lanaCardAtm.svg`